docs/mutators/lit_int.md

```diff
@@ -6,11 +6,12 @@ Integer literals like `0`, `1u8`, `5isize`
 
 ## Mutations
 
-1. replacing the literal by with a different literal
+1. replacing the literal with a different literal
```

Comment:
I think we should be able to make integer literal mutation optimistic. This would also allow us to support 128-bit types.

Reply:
I agree. The issue is that the method to get the value of the literal `value()` returns a `u64`. (https://docs.rs/syn/0.15.40/syn/struct.LitInt.html). 128-bit literals are trivially supported if their values fit inside a `u64`.
I am currently investigating what happens within `syn` in case of overflows.

Reply:
I agree. The issue is that the method to get the value of the literal `value()` returns a `u64`. (https://docs.rs/syn/0.15.40/syn/struct.LitInt.html). 128-bit literals are trivially supported if their values fit inside a `u64`.
I am currently investigating what happens within `syn` in case of overflows.

Reply:
We can get the Token::Lit instead & get the String value from there.

Reply:
During my research, I stumbled apon [your PR](https://github.com/dtolnay/syn/pull/574) towards this issue. I agree with the author of syn on the plans for the new api.
I suggest deferring this issue until `syn-1.0` has landed. As you know, this affects only literals with a value larger than `u64::max_value()` and not all values of type `u128`. This behavior is documented in [`syn::LitInt`](https://docs.rs/syn/0.15.42/syn/struct.LitInt.html).

Reply:
:+1: